manifests/blog.pp

```diff
@@ -139,7 +125,7 @@
 
   if $use_supervisor {
     supervisor::program { "ghost_${blog}":
-      command        => "node ${home}/index.js",
+      command        => "nodejs ${home}/index.js",
```

Comment:
We could make this user parameterisable, then if you get weird issues on your platform you could specify what you wanted?

Makes it easier for users to resolve issues if they hit the hell that is http://debian.2.n7.nabble.com/Node-js-and-it-s-future-in-debian-td1042211.html....